### PR TITLE
User Story 95456

### PR DESCRIPTION
**Pull request by TomArcherMsft**
[User Story 95456](https://dev.azure.com/msft-skilling/Content/_workitems/edit/)

### CODE REVIEW

**Comment by TomArcherMsft** [comment]
@grayzu @lonegunmanb @stemaMSFT The `azure-vote.yaml` file is used in the article to deploy an application.

**Review by lonegunmanb — APPROVED** [review]
Thanks @TomArcherMsft, LGTM!

**Review by lonegunmanb — APPROVED** [review]
Thanks @TomArcherMsft, LGTM!

File: quickstart/201-k8s-cluster-with-tf-and-aks/variables.tf [hunk 1]
```diff
@@ -16,27 +16,6 @@ variable "node_count" {
...
-  description = "The SKU of the Log Analytics workspace. Choose from: Free, PerNode, Premium, Standard, Standalone, Unlimited, CapacityReservation, PerGB2018"
-  default     = "PerGB2018"
-
-  validation {
-    condition     = contains(["Free", "PerNode", "Premium", "Standard", "Standalone", "Unlimited", "CapacityReservation", "PerGB2018"], var.log_analytics_workspace_sku)
-    error_message = "The Log Analytics workspace SKU must be one of the following: Free, PerNode, Premium, Standard, Standalone, Unlimited, CapacityReservation, PerGB2018"
-  }
-}
-
 variable "msi_id" {
```
Comment (grayzu) [inline quickstart/201-k8s-cluster-with-tf-and-aks/variables.tf:19]: This should be included in the readme file.

**Review by grayzu — CHANGES_REQUESTED** [review]
Thanks for the PR!

Looks mostly good. One minor change.

**Comment by stemaMSFT** [comment]
@grayzu I think after Tom's change we're just waiting on the E2E test. Can you kick it off?

**Review by lonegunmanb — APPROVED** [review]
LGTM!

**Comment by stemaMSFT** [comment]
Are we good to merge this PR?

**Comment by TomArcherMsft** [comment]
@stemaMSFT The pattern with Mark/Zijie has been that once they both have verified the code, Zijie responds with LGTM (or similar) and Mark merges. Therefore, since Zijie has given his "LGTM" comment, I would think you're okay to merge.

**Pull request merged**